cli/internal/config/config_file.go

```diff
@@ -158,6 +159,11 @@ func ReadRepoConfigFile(path turbopath.AbsoluteSystemPath, flags *pflag.FlagSet)
 	if err := repoViper.ReadInConfig(); err != nil && !os.IsNotExist(err) {
 		return nil, err
 	}
+
+	if repoViper.IsSet("remoteCacheTimeout") {
+		return nil, fmt.Errorf("remoteCacheTimeout is not a valid config option")
+	}
+
```

Comment:
I think erroring here is overkill. I believe @nathanhammond just didn't want to give users the ability to configure the timeout from the repo config. With the removal of `BindPFlag` call and no longer reading the timeout from the config object, there's no longer an API we'd need to deprecate.

Reply:
Cool I'll remove it